packages/pigeon/pigeons/proxy_api_tests.dart

```diff
@@ -0,0 +1,469 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:pigeon/pigeon.dart';
+
+enum ProxyApiTestEnum {
+  one,
+  two,
+  three,
+}
+
+/// The core interface that each host language plugin must implement in
```

Comment:
This needs to be adjusted for this class so it's not a direct copy/paste of the other comment.